schedule/yast/remote_ssh_controller.yaml

```diff
@@ -2,21 +2,24 @@
 name:           remote_ssh_controller
```

Comment:
I don't remember we made it work with MM scenarios, but I might have missed that, could you confirm it or checked? Anyway probably you wanted this file in the other commit.